README.txt

```diff
@@ -60,6 +60,37 @@ Cd to `src` and run:
     CCNAME=gcc47 gmake
     gmake install
 
+#### Building on Cygwin
```

Comment:
This looks like a good addition, but belongs in a different commit (if not PR) as it addresses a separate improvement than the subject states.